assets/less/site/rdm/elements/segment.variables

```diff
@@ -0,0 +1,2 @@
+@boxShadow: none;
```

Comment:
Disabling this globally might have undesired side effects, disable for specific case using shadowless class, use basic segment, or make selector specific